server/command.go

```diff
@@ -72,6 +74,8 @@ func (p *Plugin) executeCommand(c *plugin.Context, args *model.CommandArgs) (str
 		return p.runStartCommand(args, user)
 	case "disconnect":
 		return p.runDisconnectCommand(user)
+	case "follow_status":
```

Comment:
@aaronrothschild The code is currently using `/zoom follow_status on` as the command at the moment.

@oh6hay Can we have this in a nested subcommand like `/zoom settings follow_status on`?